ml-agents/mlagents/trainers/trainer/trainer.py

```diff
@@ -80,7 +81,7 @@ def get_max_steps(self) -> int:
         Returns the maximum number of steps. Is used to know when the trainer should be stopped.
         :return: The maximum number of steps of the trainer
         """
-        return int(float(self.trainer_parameters["max_steps"]))
+        return int(float(self.trainer_parameters.max_steps))
```

Comment:
Does this need the `int(float(_))`?  I think we should be able to manage the type coersion in the parsing logic upstream of this.

Reply:
You're right, this is already enforced during the structuring. Removed.